### PR TITLE
2

### PR DESCRIPTION
**Pull request by iurybrasileiro**

### CODE REVIEW

**Review by iurybrasileiromsilva — APPROVED** [review]

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -1,15 +1,14 @@
 name: Run Tests
```
[the remark this answers is not in the document]
Comment (iurybrasileiro) [reply .github/workflows/run-tests.yml:1]: [SKIP MESSAGE]

**Review by iurybrasileiromsilva — APPROVED** [review]

File: .github/workflows/run-tests.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (iurybrasileiro) [reply .github/workflows/run-tests.yml:1]: [SKIP MESSAGE]

**Review by iurybrasileiromsilva — APPROVED** [review]

File: .github/workflows/run-tests.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (iurybrasileiro) [reply .github/workflows/run-tests.yml:1]: [SKIP MESSAGE]

**Review by iurybrasileiromsilva — APPROVED** [review]

File: .github/workflows/run-tests.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (iurybrasileiro) [reply .github/workflows/run-tests.yml:1]: [SKIP MESSAGE]

**Review by iurybrasileiromsilva — APPROVED** [review]

File: .github/workflows/run-tests.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (iurybrasileiromsilva) [reply .github/workflows/run-tests.yml:1]: [SKIP MESSAGE]

**Pull request merged**